### ISSUE TITLE
Send Siri voice commands

### ISSUE THREAD

**Issue by manu3b1** [postlund/pyatv#170]
Hi,
Would be great to be able to send prerecorded/TTS Siri voice commands to AppleTV. That's could be much more powerful than sending button events to control the interface, as we are merely blind about the current AppleTV UI state.
Are we far from seeing that feature?
Thx.

**Comment by postlund** [comment]
This should absolutely be possible somehow and I want to implement that as well. So I would say that it's in the backlog but the development pace is quite low at the moment due to personal life. It will come one day, cannot say when though.

**Comment by manu3b1** [comment]
As you may know, the related protobuf structure can be found here:
https://github.com/edc1591/node-appletv/blob/master/src/lib/protos/SendVoiceInputMessage.proto
Now, need some sample frames to put the right fields inside...

**Comment by ldf98** [comment]
in case not aware, the iOS shortcuts app now supports opening specific apps on AppleTV by name/text,  I hadn't seen before any direct way to do this. Perhaps it presents a path to investigate for this project (which is great, thanks!)

**Comment by postlund** [comment]
> in case not aware, the iOS shortcuts app now supports opening specific apps on AppleTV by name/text, I hadn't seen before any direct way to do this. Perhaps it presents a path to investigate for this project (which is great, thanks!)

Yeah, I've heard about that as well but I have yet not been able to figure out how that works. Maybe I should dig deeper into that for a later release.

**Comment by postlund** [comment]
The way to figure it out, by the way, would be to improve the MRP proxy to make it appear in the Shortcuts app so the traffic can be sniffed and dump the message(s). In case anyone would like to dig deeper into this...

**Comment by pjsachdev** [comment]
This would be an amazing feature addition. I don't even think that the node Apple TV clients support Siri messages.

**Comment by sklever** [comment]
Any plan on release for this feature request to send Siri command as statement via pyatv?
